activity_flag/models.py

```diff
@@ -0,0 +1,39 @@
+from django.db import models
+
+# Create your models here.
+from django.contrib.auth import get_user_model
+from django.utils import timezone
+
+User = get_user_model()
+ 
+
+
+class Activity_flag(models.Model):
+
+    BLOG = 1
+    CHAT = 2
+    FORUM = 3
+    BROWSER = 4
+
+
+    Activity_Choices = [
+
+            (BLOG, 'Blog'),
+            (FORUM, 'Forum'),
+            (BROWSER, 'Browser'),
+            (CHAT, 'Chat'),
+
+        ]
+
+    activity_type_id = models.PositiveSmallIntegerField(choices=Activity_Choices) 
+    flagged_id = models.PositiveIntegerField() # will b d pk of (Blog, Forum ...) instance
```

Comment:
you didn't add foreign keys that would get either of the blog or forum instance , any reason for that

Reply:
Will be getting the instance via pk (flagged_id). No fk needed. The Activity instance will be queried with the pk when activity type is known.

Reply:
> you didn't add foreign keys that would get either of the blog or forum instance , any reason for that

That's alright?